### DISCUSSION TITLE
Hydraulischer Abgleich

### DISCUSSION THREAD

**Discussion by thiloplanz** [OpenCCU/OpenCCU#2755] [🇩🇪 Generelle Diskussionen]
Ich überlege mir, in homematic Thermostate einzusteigen, weil diese damit beworben werden, dass sie den gesetzlich geforderten hydraulischen Abgleich automatisch erledigen können, ohne dass jemand manuelle Berechnungen durchführen und neue Ventile einbauen muss.

Ich nehme mal an, dass die Thermostate dies nicht selber machen, sondern von der Kontrolleinheit entsprechend angesteuert werden.

Kann das RaspberryMatic auch oder wäre ich auf die Original CCU3 (oder gar die Cloud-Lösung :-( ) angewiesen?

**Comment by jens-maus** [comment]
Diesbzgl. unterschieden sich RaspberryMatic und CCU3 (oder die Cloud-Lösung) in keinster Weise. Es ist also vollkommen egal welche der Lösungen man nutzt – ausser das man natürlich immer Abstand von Cloud-Lösungen nehmen sollte aus heutiger Sicht!

Allerdings würde ich jetzt nicht zuviele Hoffnungen bzw. Erwartungen in das Thema "Hydraulischer Abgleich" via den eQ3/HomematicIP Heizkörperventilen/Thermostaten legen, denn auch wenn das damit beworben wird, heisst das noch lange nicht das das auch wie erwartet/beworben funktioniert. Ein hydraulicher Abgleich einer Heizungsanlage ist sehr komplex und kann systembedingt nicht einfach durch Technik am Ende der Kette (via automatischer Heizkörperventile, etc.) erledigt werden. Das was da die eQ3/HomematicIP Technik versucht als "Hydraulischer Abgleich" zu erledigen hat wirklich nichts mit einem klassischen hydraulischen Abgleich einer kompletten Heizungsanlage zu tun, denn die Heizkörperventile/Fußbodenheizungsaktoren haben ja z.B. keinen Durchflussmengenmeter und können daher nur sehr grob (anhand von Temperatur und Zeitverhältnis) schätzen wie sich das mit der Durchflussmenge in einem Raum ggf. zu einem anderen verhält. Ich würde daher nicht viel auf diese Werbeversprechen setzen.

**Comment by thiloplanz** [comment]
> Ich nehme mal an, dass die Thermostate dies nicht selber machen, sondern von der Kontrolleinheit entsprechend angesteuert werden.

Es kann durchaus sein, dass diese Annahme falsch ist.

Wie genau das System funktioniert, scheint nirgendwo dokumentiert zu sein, aber an vielen Stellen wird davon berichtet, dass die einzelnen Thermostate unabhängig voneinander über die ersten Wochen hin versuchen zu "lernen", wann welche Ventileinstellungen wie wirken. Und scheinbar (die Meinungen gehen auseinander) kommt dann am Ende ein mit dem hydraulischen Abgleich vergleichbares Ergebnis heraus. Siehe z.B. https://homematic-forum.de/forum/viewtopic.php?t=75663

Wenn das die Thermostate wirklich autark machen, dann hätte das mit der Steuereinheit nichts zu tun und würde auch mit RaspberryMatic (oder auch komplett offline) genauso gut (wertneutral) funktionieren.

**Reply by thiloplanz** [reply]
Im Umkehrschluss sollte eine intelligente Kontrolleinheit (die ja durchaus über Daten zur Aussentemperatur, Anwesenheit von Personen, geöffnete Fenster, Ventilstellungen und Temperaturen in anderen Räumen, Vorlauf- und Rücklauftemperaturen, Heizenergieverbrauch etc verfügen könnte) dann doch auch einiges an Optimierungspotential bieten.

**Comment by ivo7B4** [comment]
Ich habe soeben diesen thread hier gelesen und möchte dies noch mit meinem Verständnis ergänzen:
Der hydraulische Abgleich findet weder beim Raumthermostat, noch in der Zentrale statt. Dieser findet im Fussbodenheizungscontroller (dem Falmot c12, wired oder Funk) statt. Ich bin mir ziemlich sicher, dass dies das ist, was als Steuereinheit bezeichnet wird. Dies natürlich mit den Informationen der Raumthermostate, Ventilpositionen, ...
Wenn mehrere Falmot's verwendet werden, wäre es zusätzlich aus meiner Sicht von Vorteil, wenn dies dann über die Zentrale koordiniert werden würde - was aber nicht erfolgt, soweit ich weiss. Ich bin damit aber dennoch ziemlich zufrieden (mehrere Falmots in Verwendung, jedoch max. einer pro Stockwerk).
